skopt/space.py

```diff
@@ -158,25 +163,35 @@ def __init__(self, low, high, prior="uniform"):
             - If `"log-uniform"`, points are sampled uniformly between
               `log10(lower)` and `log10(upper)`.`
         """
-        self._low = low
-        self._high = high
+        self.low = low
+        self.high = high
         self.prior = prior
 
         if prior == "uniform":
-            self._rvs = uniform(self._low, self._high - self._low)
+            self._rvs = uniform(self.low, self.high - self.low)
             self.transformer = _Identity()
 
         elif prior == "log-uniform":
             self._rvs = uniform(
-                np.log10(self._low),
-                np.log10(self._high) - np.log10(self._low))
+                np.log10(self.low),
+                np.log10(self.high) - np.log10(self.low))
             self.transformer = _Log10()
 
         else:
             raise ValueError(
                 "Prior should be either 'uniform' or 'log-uniform', "
                 "got '%s'." % self._rvs)
 
+    def __eq__(self, other):
+        return (type(self) is type(other)
+                and self.low == other.low
+                and self.high == other.high
+                and self.prior == other.prior)
```

Comment:
Worth using `np.allclose` to compare the prior which will be weird floating point numbers?